code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -805,4 +1087,7 @@
 	locked = FALSE
 	INVOKE_ASYNC(src, PROC_REF(open))
 
+/obj/structure/closet/open
+	opened = 1
+
```

Comment:
icon_state = "closetopen" is not necessary because `Initialize()` calls `update_appearance()` which will set the icon state based on the `opened` variable. I'll change it to true though